app/forms/two_factor_options_form.rb

```diff
@@ -42,4 +42,9 @@ def update_otp_delivery_preference_for_user
       selection.find { |element| %w[voice sms].include?(element) } }
     UpdateUser.new(user: user, attributes: user_attributes).call
   end
+
+  def phone_selected?
+    return false if !IdentityConfig.store.select_multiple_mfa_options
+    selection.include?('phone') || selection.include?('voice') || selection.include?('sms')
```

Comment:
I'm sorta confused by what selections this class is expecting to receive. In practice, we only offer the user a single "phone" option when configuring MFA, so when would it be "voice" or "sms"?

Reply:
I think that's for editing the users's selection later. After they change their preference, we still need to enforce the requirement for alternatives to phone, sms, and voice.